src/ima-state-machine.js

```diff
@@ -220,8 +220,10 @@ function onAdClicked(options: Object, adEvent: any): void {
   if (this._currentAd.isLinear()) {
     if (this._isVideoAd()) {
       this._maybeIgnoreClickOnAd();
-      if (this._stateMachine.is(State.PLAYING)) {
+      if (this._stateMachine.is(State.PLAYING) && !this.player.isLive()) {
```

Comment:
need to pause on mobile like in [dai](https://github.com/kaltura/playkit-js-ima-dai/blob/master/src/ima-dai.js#L704)

Reply:
tested on android - is automatically paused as expected